cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+                    react_sent_by_user_modifying = False
+                    manual_timeout = time.time() + 60
+                    while not react_sent_by_user_modifying:
+                        reaction, reaction_user = await self.bot.wait_for(
+                            'reaction_add',
+                            check=reaction_check_any_user,
+                            timeout=60)
+                        if reaction_user == user:
+                            react_sent_by_user_modifying = True
+                        else:
+                            if time.time() > manual_timeout:
+                                raise asyncio.TimeoutError
+
+                except asyncio.TimeoutError:
+                    if proposals:
+                        title = 'The modification of the custom reaction proposal timed out. ' \
+                                'Returning to list of reaction proposals...'
+                    else:
+                        title = 'The modification of the custom reaction timed out. ' \
+                                'Returning to list of current reactions...'
+                    embed = discord.Embed(title=title)
+                    await message.edit(embed=embed)
+                    conn.close()
+                    time.sleep(5)
+                    current_options.clear()
+                    await message.clear_reactions()
+                    return
+
+                current_options.clear()
+                await message.clear_reactions()
+                if reaction is None:
+                    return
+                elif reaction.emoji == "0️⃣":
+                    if anywhere == 0:
+                        if proposals:
+                            title = "Successfully kept current option! Returning to list of reaction proposals..."
+                        else:
+                            title = "Successfully kept current option! Returning to list of current reactions..."
+                        embed = discord.Embed(title=title)
+                        embed.set_footer(text="Modified by {}.".format(user),
+                                         icon_url=user.avatar_url)
+                        await message.edit(embed=embed)
+                        conn.close()
+                        time.sleep(5)
+                    else:
+                        t = (0, custom_react_id)
+                        c.execute(
+                            'UPDATE CustomReactions SET Anywhere = ? WHERE CustomReactionID = ?',
+                            t)
+                        conn.commit()
+                        self.rebuild_reaction_list()
+                        self.rebuild_proposal_list()
+                        if proposals:
+                            title = "Option successfully modified! Returning to list of current reaction proposals..."
+                        else:
+                            title = "Option successfully modified! Returning to list of current reactions..."
```

Comment:
duplicated string literals – use variables and format as needed

I think this string came up before as well

Reply:
See https://github.com/idoneam/Canary/pull/273#issuecomment-656869764